docs/documentation/admin/admin_company.md

```diff
@@ -9,18 +9,26 @@ import Highlight from '@theme/Highlight';
 <img alt="design" class="img_sizing" src={useBaseUrl('img/design/Company.svg')} />
 
 ## Overview {#company-overview}
-<img alt="" src={useBaseUrl('img/admin_company.png')} width= "20%" height= "20%" align="left"/>
-The company is where the created users belong. For now, the user can only be related to one company, but in the future it could be more than one. <br/> In this section you can only edit aspects of the company.
-<br clear="left" />
+The _company_ is where the created users belong. For now, the user can only be related to one company, but in the future it could be more than one. 
```

Comment:
The __company__ represents your organization and relates all your databases, users, workflow, bot, etc.